lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -203,13 +203,14 @@ def main():
         with tables.open_file(args.output_file, mode='a') as output:
             params = output.root[dl1_params_lstcam_key].read()
 
-            for tab in [output.root[dl1_params_lstcam_key], output.root[dl1_images_lstcam_key]]:
+            tabs = [output.root[dl1_params_lstcam_key]]
+            if not args.noimage:
+                tabs.append(output.root[dl1_images_lstcam_key])
+            for tab in tabs:
                 # need container to use lstchain.io.add_global_metadata and lstchain.io.add_config_metadata
                 for k, item in metadata.as_dict().items():
                     tab.attrs[k] = item
                 tab.attrs["config"] = str(config)
```

Comment:
I think that the image table is not produced during the execution of this script, but we are just copying the image table that has been produced with `lstchain_data_r0_to_dl1.py` script, so the image table has already its own provenance config stored in the execution of a previous script. So I would not reassign  the config to the image table. 

I would just **not consider** the image table, I would remove the loop in tabs and consider only the parameters table, as below:

```python
    # need container to use lstchain.io.add_global_metadata and lstchain.io.add_config_metadata
    for k, item in metadata.as_dict().items():
        output.root[dl1_params_lstcam_key].attrs[k] = item
        output.root[dl1_params_lstcam_key].attrs["config"] = str(config)
```